### PR TITLE
bump versions

### PR DESCRIPTION
**Pull request by giggsoff**
Sync version of EVE 

Signed-off-by: Petr Fedchenkov <giggsoff@gmail.com>

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -20,7 +20,7 @@ COPY . /adam/src
 ARG GOOS=linux
 ARG GOARCH=amd64
 
-RUN go build -o /adam/bin/adam main.go
+RUN go build -ldflags "-s -w" -o /adam/bin/adam main.go
```
Comment (deitch) [inline Dockerfile:23]: Why this?

**Review by deitch — COMMENTED** [review]
Most of this looks good, but what do the ldflags buy us?

File: Dockerfile (hunk 1 above)
Comment (giggsoff) [reply Dockerfile:22]: They [remove debug info](https://golang.org/cmd/link) and reduce the size of image.

Comment (deitch) [reply Dockerfile:22]: They remove some, but [not everything](https://github.com/golang/go/issues/19226). Do we care? We aren't distributing the image in a production capacity. How much difference does it make in size, before and after?

File: Dockerfile [hunk 2]
```diff
@@ -18,9 +18,8 @@ RUN go mod download
 COPY . /adam/src
 
 ARG GOOS=linux
-ARG GOARCH=amd64
```
Comment (giggsoff) [inline Dockerfile:21]: Also, seems, that we need to remove this line now. We do not use build-args, but we have action to build with defined platform (linux/arm64).

File: Dockerfile (hunk 1 above)
Comment (giggsoff) [reply Dockerfile:22]: 18 vs 13 MB for binary

Comment (deitch) [reply Dockerfile:22]: I really don't care about the 5MB in size; do we use adam anywhere (on embedded systems) where it really matters? I guess if it already is in here, we can leave it.

File: Dockerfile (hunk 2 above)
Comment (deitch) [reply Dockerfile:21]: Does having this line _break_ anything?

File: Dockerfile (hunk 1 above)
Comment (giggsoff) [reply Dockerfile]: Well, removed for now.

File: Dockerfile (hunk 2 above)
Comment (giggsoff) [reply Dockerfile:21]: Yes, it break building for arm64 inside action (we compile amd64 binary in all cases).

Comment (deitch) [reply Dockerfile:21]: Understood.

Considering how quick and painless CI for adam is, we should not intermix these. One PR for bumping alpine, one for adding/removing build ARGs. These are unrelated changes (just like the ldflags is an unrelated change).

Please remove it from here, keep this PR really simple, we can merge it in, and then we can open a separate small PR for that.

Comment (giggsoff) [reply Dockerfile]: Done and created new one #70

**Review by deitch — APPROVED** [review]

**Comment by deitch** [comment]
Thanks!

**Pull request merged**